src/rime/lever/deployment_tasks.cc

```diff
@@ -634,6 +634,9 @@ bool CleanOldLogFiles::Run(Deployer* deployer) {
 
   int removed = 0;
   for (const auto& dir : dirs) {
+    // avoid iteration on non-existing directory, which may cause error
+    if (!fs::exists(path(dir)))
```

Comment:
this is `rime::path`, it takes the path string as UTF-8 encoded.

`google::GetLoggingDirectories()` maybe returns native encoded path. If so, this check can skip actually existing paths which contain Unicode characters.

Reply:
updated